scripts/run-targeted-tests.js

```diff
@@ -0,0 +1,86 @@
+const WRAPPERS = [
+  'angular-handsontable',
+  'react-handsontable',
+  'vue-handsontable',
+];
+const FULL_TEST_BRANCHES = [
+  'master',
+  'develop',
+  'release/',
+];
+
+const { spawnSync, execSync } = require('child_process');
+
+const currentBranch = execSync('git rev-parse --abbrev-ref HEAD', { encoding: 'utf8' }).replace(/(\n)/gm, '');
+const filesModifiedInLastCommit = execSync('git log --name-only --oneline HEAD^..HEAD', { encoding: 'utf8' })
+  .split('\n');
+let hasErrors = false;
+
+const spawnCommand = (command) => {
+  if (hasErrors) {
+    return;
+  }
+  const spawnedCommand = spawnSync('npm', command.split(' '), {
+    stdio: 'inherit'
+  });
+
+  if (spawnedCommand.status) {
+    hasErrors = true;
+  }
+
+  return !hasErrors;
+};
+
+filesModifiedInLastCommit.shift();
+filesModifiedInLastCommit.pop();
+
+const fullTestBranchMatch = currentBranch.match(
```

Comment:
I think that the regex in its current shape could be improved. Now it passes the names like `masterrrr` or `master-hmm-haa`.

I can suggest to replace the `FULL_TEST_BRANCHES` const with regex e.g. 
```js
const fullTestBranchRegex = new RegExp('^(master|develop|release\/.{5,})$');
```
and, then the testing can be simplified to something like:
```js
fullTestBranchRegex.test(currentBranch);
```